client/app/account/user/credentials/user-credentials.controller.js

```diff
@@ -0,0 +1,42 @@
+angular.module("UserAccount.controllers").controller("UserAccount.controllers.credentials", [
+    "$scope",
+    "$q",
+    "UserAccount.services.credentials",
+    "Alerter",
+    function ($scope, $q, UseraccountCredentialsService, Alerter) {
```

Comment:
You didn't use `$q` service.

I encourage you to use `$q.all([])` where you map your `credentialsIds` to the promise you want to execute.

However in your `init` function, what I suspect is the amount of calls will grow up exponentially and in my opinion it will be a better to solution to use a pagination mechanism.

Small detail but feel free to write your controller as an entire ES6 `class` if you are comfortable with. :)